filebeat/docs/getting-started.asciidoc

```diff
@@ -99,9 +99,33 @@ To see the full list of variables for a module, see the documentation under
 
 include::{libbeat-dir}/shared/config-check.asciidoc[]
 
+[float]
+[[setup-application-logs]]
+=== Step 4: Ingest application logs
```

Comment:
Makes sense but I think step 3 should then be marked optional, too.

Reply:
When we created the quick start, we decided to focus on the quick getting started path with modules. We wanted users to be able to see data quickly. For custom logs, we decided to send users to the section about configuring the input manually. 

I agree this is not a good user experience. We dump users into the input config docs, where we don't really tell them everything they need to do to ingest their logs. We don't really talk about how to parse the logs at all. The plan was to add a topic about ingesting custom logs, but we just haven't done that yet.

The problem with adding that info to the QS is that it makes the setup experience a lot more challenging, and it introduces concepts (like ingest node and ECS) that we can't assume new users know.

 I think we need to:
 
1) Create a topic under the "how to" section (something like "Ingest application logs" or "Ingest custom logs") where we explain best practices *and* provide step-by-step instructions to guide users through ingesting application logs.
2) Make it clear in the quick start that the steps apply only to common log formats for which we have modules. (I do have a sentence, but it might be too subtle.) Plus we need to add a pointer to the related how-to topic.

WDYT?

If you feel strongly about having this info here, then the section needs to walk users through step-by-step instructions. It also needs to be clear that the step only applies if you're not using modules to parse the logs.

Reply:
Instead of turning this into a how-to guide, would it make sense to have two quickstarts? One for modules, and one for application logs?

Reply:
@bmorelli25 That's a good idea. I think both are valid "getting started" experiences. 

Not sure about a 3rd getting started. Wouldn't a topic about ingesting custom logs cover application logs, too? 

@felixbarny If you want to move forward with a second getting started, let us know. We can help you set up the basic structure. We might need to add some new command widgets (it's a manual process right now). I think it would be quick to set something up.

Reply:
Just had another idea. Threw up a draft PR here: https://github.com/elastic/beats/pull/23371. Take a looksie.